r/tests/testthat/test-trackplots.R

```diff
@@ -180,44 +192,54 @@ test_that("trackplot_coverage doesn't crash", {
       atac_reads = as.integer(seq(from=0, to=10, length.out=length(cell_names))),
       cell_type = sample(c("T", "B", "NK"), size = length(cell_names), replace=TRUE)
     )
-    region <- tibble::tibble(
+    # general region
+    region1 <- tibble::tibble(
         chr = "chr15",
         start = 30000000,
         end = 52806155,
     )
-    expect_no_condition(
-        ggplot2::ggplot_build(trackplot_coverage(
-              frags,
-              region=region,
-              groups=frags_metadata$cell_type,
-              cell_read_counts=frags_metadata$atac_reads,
-              bins=500
-            )
-        )
+    # for small motifs
+    region2 <- tibble::tibble(
+        chr = "chr15",
+        start = 31283779,# 31277136,
+        end = 31283870#31322748
```

Comment:
Shouldn't have commented-out code left in the tests

Reply:
Oops!